### ISSUE TITLE
Retrieve both temperatures the thermostat monitors

### ISSUE THREAD

**Issue by rct** [alex-0103/warmup4IE#5]
Would it be possible to retrieve both temperatures that the thermostat monitors?  

They have a thermocouple "probe" that usually gets the temperature of the floor.  There is also a sensor in the thermostat that is displayed as the "air temperature".

FWIW, I took a look at the [SmartThings code](https://community.smartthings.com/t/beta-warmup-connect-warmup-4ie-wifi-thermostat/68296). The screen shots show the air temperature listed next to the run mode. Skimming the Groovy code, I don't see where it is picking that up.  But just skimming I don't see the same sort of GraphQL query either.

Thanks.

**Comment by foxy82** [comment]
To do this you need to change the query in  update_room

`body = {
    "query": "query QUERY{ user{ currentLocation: location { id name rooms{ id roomName runModeInt targetTemp currentTemp thermostat4ies {minTemp maxTemp airTemp}}  }}  } "
}`

or even better to have floor, air and what you are heating....

`body = {
     "query": "query QUERY{ user{ currentLocation: location { id name rooms{ id roomName runModeInt targetTemp currentTemp thermostat4ies {minTemp maxTemp airTemp floor1Temp floor2Temp heatingTargetInt}}  }}  } "
}`

where heatingTargetInt returns 0 for targeting floor temp and 1 for targeting air temp.

**Comment by foxy82** [comment]
Created pull request to get them and everything else that might be useful that I found on the API https://github.com/alex-0103/warmup4IE/pull/6

**Comment by rct** [comment]
Not sure if I should close this here.  This is now working correctly in the master branch of the new repo - [ha-warmup/warmup](https://github.com/ha-warmup/warmup)

**Comment by ploner** [comment]
> em and everything else that might be useful

Was the pull request accepted, and one can retrieve both temperatures?
If yes, thank you!

**Comment by artmg** [comment]
Unfortunately the maintainer of this project has been unable to act on any pull requests, and this project appears dormant since five years ago. If you are trying to use Warmup products with Home Assistant, you might consider a fork of this project, https://github.com/ha-warmup/warmup - good luck!

**Comment by ploner** [comment]
Honestly, I use IOBroker, plus custom Python scripts with cron.
The original python library worked for me. Is your fork also packaged and
usable?
Thank you

**Comment by foxy82** [comment]
It is on PyPi - so best to give it a go:

https://pypi.org/project/warmup4ie/

**Comment by artmg** [comment]
No, sorry, @ploner that fork has not addressed any shortcomings in the Warmup4IE pypi package itself, merely in the wrapper for the HA integration. Unfortunately, I have not heard anyone step forward yet to manage the Warmup4IE pypi package in place of this earlier maintainer. Perhaps you might try a fork yourself, then you would have the rights to add the PR mentioned above
